app/pages/organisation/ui.test.js

```diff
@@ -126,8 +126,8 @@ test('should render edit org button', async (t) => {
     .expect(editOrgButton.exists).ok()
     .expect(await extractInnerText(editOrgButton)).eql(content.editOrgButtonText)
     .expect(editOrgButton.hasClass('nhsuk-u-margin-bottom-9')).ok()
-    // TODO: Change when edit organisation implemented
-    .expect(editOrgButton.getAttribute('href')).eql('#');
+    .expect(editOrgButton.getAttribute('href')).eql('/organisations/org1/edit');
+  // TODO: click through
```

Comment:
We do have the edit page now though. Can this be done now?

Reply:
yes I am doing this now 😃